pkg/kind/kind.go

```diff
@@ -85,8 +88,11 @@ func createKindCluster() error {
 	time.Sleep(10 * time.Second)
 
 	clusterWait := exec.Command("kubectl", "wait", "pod", "--timeout=-1s", "--for=condition=Ready", "-l", "!job-name", "-n", "kube-system")
-	if err := clusterWait.Run(); err != nil {
+	if out, err := clusterWait.CombinedOutput(); err != nil {
+		fmt.Println(string(out))
```

Comment:
If you're going to print it out even if it succeeds and not just on failure, it's probably easier/better to just do cmd.Stdout = os.Stdout and cmd.Stderr = os.Stderr. I'd also probably pull out a util to run and print commands at this point since we're repeatedly making changes in multiple places

Reply:
It's probably a bit much to print the output when things succeed (I'd imagine new users don't really need to see all that)

Reply:
let's drop the else block then?

Reply:
done